pyproject.toml

```diff
@@ -57,5 +57,11 @@ joblib = "^1.4.2"
 itde = 'exasol_integration_test_docker_environment.main:main'
 
 
+[[tool.mypy.overrides]]
+module = [ "luigi.*", "docker.*", "humanfriendly", "configobj", "toml", "netaddr", "joblib.testing", "networkx",
+            "fabric", "requests", "pyexasol", "paramiko.ssh_exception", "six", "jsonpickle", "exasol.toolbox.*",
```

Comment:
Ok it seems we need the py.typed file in the toolbox 
https://blog.whtsky.me/tech/2021/dont-forget-py.typed-for-your-typed-python-package/

@Nicoretti could you check this

Reply:
Created https://github.com/exasol/python-toolbox/issues/293